docs/modules/user-guide/pages/adding-kubernetes--or-openshift-component-to-a-devfile.adoc

```diff
@@ -0,0 +1,6 @@
+:description: Adding kubernetes or openshift component to a devfile
```

Comment:
Review the capitalization of product names throughout.

Reply:
We consistently don't capitalize. if you think we ought to, I'm happy to, but I'd rather do so in a separate PR, specifically intended for capitalization.

Reply:
@jc-berger We should correctly capitalize product names. Official product names, including their capitalization, are usually available from an organization's trademark guidelines, such as https://www.redhat.com/en/about/brand/standards/trademarks

When you identify a small error like this, it is usually best to fix it right away. This is a well-known software engineering practice, sometimes called "fix as you go." Deferring the fix allows the error to go live and become part of the customer experience.

Reply:
Sorry. Disregard. I see now that we're talking about a literal `kubernetes` or `openshift` component. I didn't see the full context when I made this comment.